homeassistant/components/tellduslive/sensor.py

```diff
@@ -95,7 +95,7 @@
     SENSOR_TYPE_WATT: SensorEntityDescription(
         key=SENSOR_TYPE_WATT,
         name="Power",
-        native_unit_of_measurement=POWER_WATT,
+        native_unit_of_measurement=UnitOfPower.WATT,
```

Comment:
We could add device class here.
Before I do that, should this be `IRRADIATION_WATTS_PER_SQUARE_METER` like other weather entities?

Reply:
Without response from the code owner, I think it's safer to assume to the current unit is correct, and simply add the device class.